src/workflow.ts

```diff
@@ -80,7 +80,7 @@ export interface WorkflowStatus {
   readonly error?: unknown; // The error thrown by the workflow, if any.
   readonly input?: unknown[]; // The input to the workflow, if any.
 
-  readonly request?: HTTPRequest; // The parent request for this workflow, if any.
+  readonly request?: object; // The parent request for this workflow, if any.
```

Comment:
I realize this decision flows thru a bunch of other changes, but do we want to force request to be an object instead of unknown (which could be a primive value like number or string)? Or is it important for some reason this is an object?